stdlib/public/TensorFlow/CMakeLists.txt

```diff
@@ -52,7 +52,6 @@ add_swift_target_library(swiftTensorFlow ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_
   "${SOURCES}"
 
   INCORPORATE_OBJECT_LIBRARIES swiftCTensorFlow
-  TARGET_SDKS OSX LINUX
   PRIVATE_LINK_LIBRARIES "${TF_LIBRARIES}"
   SWIFT_MODULE_DEPENDS SwiftOnoneSupport
```

Comment:
Unrelated: I think `SWIFT_MODULE_DEPENDS SwiftOnoneSupport` is obsolete and can be removed.

Reply:
Yeah, that sounds about right.  I'll fix that up in a follow up change.